packages/create-llama/templates/components/loaders/python/web/loader.py

```diff
@@ -1,5 +1,5 @@
 import os
-from llama_index import VectorStoreIndex, download_loader
+from llama_index.core.readers.download import download_loader
```

Comment:
no package for `WholeSiteReader`?

Reply:
We are using `download_loader`, that function changed behavior that now run the `pip install` command so the package will be installed automatically. 
But in newer commits, i also tested our addPoetryDependencies function for tools and vectordbs so let me consider moving this wholesitereader to that function instead of using the download one, it'll be clearer and more consistent with other integration

Reply:
I just removed the download_loader to use our installation instead. The dependencies for integrations should be declared in the code if it's not included in default llama_index package. Can check them out in the `getAdditionalDependencies` function.